targets/TARGET_ARM_SSG/TARGET_CM3DS_MPS2/objects.h

```diff
@@ -70,6 +70,11 @@ struct analogin_s {
     uint16_t ctrl_register;  /* Control bits with the channel identifier */
 };
 
+/* This structure is not used by the HAL implementation. */
+struct flash_s {
+    uint32_t not_used;
```

Comment:
I guess this field is defined for compatibility with IAR and ARMC5 but for consistency shouldn't it be a `uint8_t` like in `struct trng_s` ?

Reply:
Use uint8_t for consistent.